adapters/aceex/aceex.go

```diff
@@ -84,6 +84,7 @@ func (a *adapter) MakeRequests(
 		Body:    reqJSON,
 		Uri:     url,
 		Headers: getHeaders(openRTBRequest),
+		ImpIDs:  []string{openRTBRequest.Imp[0].ID},
```

Comment:
Why is this returning the id of just the first imp? The full request is passed to the adapter. I see the same pattern in other adapters.

Reply:
Passed all imps in request for all the bidders where I passed first imp.